client/src/components/SideMenu/styled/styled.tsx

```diff
@@ -19,27 +20,53 @@ export const NavbarContainer = styled.div<IMenuProps>`
   top: 0;
 `;
 export const NavbarItem = styled(NavLink)<IMenuProps>`
+  border-right: 1px solid rgba(96, 96, 96, 0.273);
   font-size: 20px;
-  margin: 15px 0;
-  padding-left: 15px;
+  padding-left: ${({ padding }) => padding};
   cursor: pointer;
-  height: 35px;
+  height: 45px;
   display: flex;
   align-items: center;
-  justify-content: space-between;
-  width: 220px;
-
   text-decoration: none;
-  color: white;
+  color: rgb(127, 126, 143);
   font-size: 20px;
   font-weight: 200;
-  background-color: ${({ pickedid, itemid }) =>
-    pickedid === itemid ? "#aaaaaa52" : "transparent"};
   &:hover {
-    background-color: #ffffff45;
+    background-color: #aaaaaa52;
+    color: black;
+  }
+  span {
+    padding-left: 15px;
+    text-align: start;
+    width: 170px;
+  }
+`;
+
+export const Title = styled.div<IMenuProps>`
+  padding: 9px 0 9px 15px;
+  &:hover {
+    background-color: #aaaaaa52;
     color: black;
   }
+`;
+
+export const TitleNavbarSubItem = styled.div<IMenuProps>`
+  border-right: 1px solid rgba(96, 96, 96, 0.273);
+  font-size: 20px;
+  cursor: pointer;
+  overflow: hidden;
+  height: ${({ pickedId, itemId, height }) =>
+    pickedId === itemId ? height : "45px"};
```

Comment:
can you make a props isItemSelected
and use as isItemSelected ? height : '45px'